nissan_2017.dbc

```diff
@@ -86,7 +86,7 @@ BO_ 389 Steering: 8 XXX
  SG_ LKAS_Torque : 47|8@0+ (1,0) [0|255] "" XXX
  SG_ CRC : 63|8@0+ (1,0) [0|127] "" XXX
  SG_ Angle : 23|18@0+ (-0.01,1310) [0|262143] "" XXX
- SG_ DriverTorque : 7|12@0+ (1,-2047) [0|4095] "" XXX
+ SG_ DriverTorque : 7|12@0+ (-1,2047.5) [0|4095] "Ncm" XXX
```

Comment:
what's `Ncm`? :D

Reply:
also, the offset should be an integer. 2047 or 2048?

Reply:
Ncm = Nm * 100. Should I scale this to Nm?

Reply:
yeah, set the factor to 0.01